contributors.txt

```diff
@@ -0,0 +1,57 @@
+certificate of origin by signing on the bottom with their github
+userid, full name, email address (you can obscure your e-mail, but it
+must be computable by human), and date.
+
+By signing this agreement, you are warranting and representing that
+you have the right to release code contributions or other content free
+of any obligations to third parties and are granting Henk Westhuis and
+GitExtensions project contributors, henceforth referred to as The GitExtensions
+Project, a license to incorporate it into The GitExtensions Project 
+or related works under the BSD license. You understand that The GitExtensions
+Project may or may not incorporate your contribution and you warrant 
+and represent the following:
+
+1. I am the creator of all my contributions. I am the author of all
+   contributed work submitted and further warrant and represent that
+   such work is my original creation and I have the right to license
+   it to The GitExtensions Project for release under the 3-clause BSD
+   license. I hereby grant The GitExtensions Project a nonexclusive,
+   irrevocable, royalty-free, worldwide license to reproduce,
+   distribute, prepare derivative works, and otherwise use this
+   contribution as part of the GitExtensions project, associated
+   documentation, books, and tools at no cost to The GitExtensions Project.
+
+2. I have the right to submit. This submission does not violate the
+   rights of any person or entity and that I have legal authority over
+   this submission and to make this certification.
+
+3. If I violate another's rights, liability lies with me. I agree to
+   defend, indemnify, and hold The GitExtensions Project and GitExtensions 
+   users harmless from any claim or demand, including reasonable attorney
+   fees, made by any third party due to or arising out of my violation
+   of these terms and conditions or my violation of the rights of
+   another person or entity.
+
+4. I understand and agree that this project and the contribution are
+   public and that a record of the contribution (including all
+   personal information I submit with it, including my sign-off) is
+   maintained indefinitely and may be redistributed consistent with
+   this project or the open source license indicated in the file.
+
+I have read this agreement and do so certify by adding my signoff to
+the end of the following contributors list.
+
+CONTRIBUTORS:
+
+YYYY/MM/DD, github id, Full name, email
+2019/01/23, spdr870, Henk Westhuis, henk.westhuis@gmail.com
+2019/01/23, KindDragon, Arkady Shapkin, arkady.shapkin@gmail.com
+2019/01/23, jbialobr, Janusz Białobrzewski, jbialobr@o2.pl
+2019/01/23, RussKie, Igor Velikorossov, russkie@gmail.com
+2019/01/23, gerhardol, Gerhard Olsson, gerhard.nospam@gmail.com
+2019/01/23, drewnoakes, Drew Noakes, git@drewnoakes.com
+2019/01/23, amaiorano, Antonio Maiorano, amaiorano@gmail.com
+2019/01/23, sharwell, Sam Harwell, sam@tunnelvisionlabs.com
```

Comment:
📝 We should probably all sign this individually

Reply:
At the very least everyone should re-accept this PR post this change.

Reply:
Yes, that was my plan.
I hope you're all ok with it.